frontend/src/components/object/ObjectList.vue

```diff
@@ -54,62 +54,76 @@ const listObjects = async () => {
   }
 };
 
+// const updateBreadcrumb = async () => {
+//   try {
+//     const bucket = await bucketStore.getBucketInfo(route.query.bucketId as string);
+//     navStore.replace('__listObjectsDynamic', bucket?.bucketName ?? 'Unknown bucket');
+//   } catch (error: any) {
+//     toast.add({ severity: 'error', summary: 'Unable to load bucket information.', detail: error, life: 5000 });
+//   }
+// };
+
 // Download
 const multiSelectedObjectIds = computed(() => {
   return multiSelectedObjects.value.map((o) => o.id);
 });
 
 // Get the user's list of objects
 onMounted(() => {
+  // Removed for now
+  // updateBreadcrumb();
```

Comment:
Same here - wondering if we need to keep the commented code?

Reply:
For this one, we want to revisit breadcrumbs at a later date, so this should be preserved.